app/src/main/AndroidManifest.xml

```diff
@@ -54,62 +54,62 @@
             android:windowSoftInputMode="adjustResize"/>
 
         <activity android:configChanges="orientation|screenSize"
-                  android:launchMode="singleTask"
-                  android:name="org.mifos.mobile.ui.activities.PassCodeActivity"
-                  android:windowSoftInputMode="adjustResize"/>
+            android:launchMode="singleTask"
```

Comment:
There's no change in this file, why format & add it in a commit? 
Please remove this file from the commit.